test/system/groups_test.rb

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Can you please add a test which tests the following condition:

1) A `Name` that isn't duplicate of an existing group and `Path` that already exists for a group

Reply:
Please see cc8bd1e7e8d81a609bedee8e118d728a65e5882e.

Reply:
Awesome thanks!